### ISSUE TITLE
Drop wa/...alpha nodes as inscriptions

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#469]
All macrogenesis idnos from wa which have an `alpha` at the end are inscriptions.
Plus, these may be highly ambiguous, since `2_H.alpha` refers to many different parts of inscription. (The latter may be part of what @sandrakrause is already doing; please open an adequate issue in faustedition/xml if this is the case.)

**Comment by thvitt** [comment]
What exactly should I do here?

**Comment by thvitt** [comment]
Nothing is dropped.

There are a number of automatic matching rules in order to find a matching witness, generated from stuff we discussed previously, starting here in the source code:
https://github.com/faustedition/faust-macrogen/blob/1c22e6b1c735574a51165e82b1471d0c74677868/uris.py#L264
If no resolution is found, the node is used without being directly linked to something in the edition. We can discuss this later.

There is an additional 'adoption' rule for cases where there is an 'orphan' witness X in the edition which is not mentioned in the macrogenesis data, but there are inscriptions X/i_something in the macrogenesis data. In this case an artificial edge between the two is created.

You can identify what happens with the references in the [references table](http://dev.faustedition.net/macrogenesis/refs): 
- gray lines are items from the edition which could not be linked to the graph
- nodes that have no link in the _Typ/Edition_ column could not be associated with items in the edition (but may be linked using assertions/edges!)

**Comment by gerritbruening** [comment]
Ja, kann man im Grunde so sagen: `faust://inscription/wa/...`. Aber worauf sich das jeweils bezieht, müsste erst mühsam recherchiert werden. Es kann sich z.B. auf [das hier oben unter dem Zettel](http://dev.faustedition.net/document?sigil=2_III_H.2_1&page=5&view=facsimile&layer=1#l8490) beziehen.
Ich denke mal, mit denen konnte nur deswegen nichts angefangen werden, weil die Grund-Sigle schon ambig ist.

**Comment by thvitt** [comment]
Das [automatische Handling für `faust://…/wa/…/alpha`](https://github.com/faustedition/faust-macrogen/blob/1c22e6b1c735574a51165e82b1471d0c74677868/uris.py#L272-L279) ist, dass versucht wird, den Teil vor dem /alpha als Zeugen zu Identifizieren. War das erfolgreich, so ist das Ergebnis die _Inskription alpha von …_, ansonsten gibt’s eine [Warnung im Log](http://dev.digital-humanities.de/ci/view/Faust/job/faust-macrogen/lastBuild/consoleFull):

```
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_II_H.31_alpha (wa_faust: 2_II_H.31_)
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2_alpha (wa_faust: 2_III_H.2_)
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.IIIalpha (2 III H.3a: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.IIIalpha (2 III H.3a: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
WARNING  uris     Could not fix WA pseudo inscription candidate faust://document/wa/2_III_H.2alpha (2 III H.2: … (2))
```

Die Fälle mit `2 III H.2: … (2)` und `2 III H.3a: … (2)` in der Meldung sind besonders problematisch, da hier schon die Sigle ohne Alpha auf jeweils zwei Zeugen in unserer Edition verweist. 

Bei den anderen Fällen kann man vielleicht überlegen, ob man sagt: _Der Gesamtzeuge ist erst fertig, wenn alle Inskriptionen fertig sind_, und deswegen Kanten _Inskription … von X_ → _X_ einzieht. Müsste mal prüfen, ob das zu Konflikten führt. Wäre diese Semantik konsistent mit euren Gedanken beim Anlegen der Makrogenesedaten, @gerritbruening?

**Comment by gerritbruening** [comment]
Das könnte wohl am ehesten @dietmarpravida sagen.

**Comment by dietmarpravida** [comment]
das müsste mir mal etwas ausführlicher erklärt werden

**Comment by thvitt** [comment]
wird hier nochwas von mir erwartet? nö oder?

**Comment by dietmarpravida** [comment]
ummeilen?
